src/components/Challenges/CodeChallenge/CodeChallenge.tsx

```diff
@@ -46,6 +46,15 @@ export const CodeChallenge = ({
 
   useEffect(resetChallenge, [id]);
 
+  // passes test when control & enter keys are pressed together
+  const passTestsOnKeypress = () => {
+    window.addEventListener('keypress', (e:KeyboardEvent) => {
+      ((e.ctrlKey || e.metaKey) && e.code === "Enter") ? runTests() : null;
+    })
```

Comment:
https://reactjs.org/docs/hooks-reference.html#cleaning-up-an-effect
This is what I mean by cleanup